superset/assets/src/dashboard/components/SliceHeader.jsx

```diff
@@ -109,6 +109,7 @@ class SliceHeader extends React.PureComponent {
                 : '')
```

Comment:
@mistercrunch the line above this allows you to click on the title to edit it from the dashboard ^^^

Reply:
oh nice! Never mind my comment then!